### ISSUE TITLE
Couldn't determine Mocha version

### ISSUE THREAD

**Issue by gcirone** [michaelleeallen/mocha-junit-reporter#87]
Hi,

I have issue with cypress when the project is stored in a different folder then root, for example:

`cypress run --project=apps/site-e2e`

Deps installed:
```
"mocha": "6.1.4",
"mocha-junit-reporter": "1.23.0",
```

I presume the problem is because the packages json path start with `./`

https://github.com/michaelleeallen/mocha-junit-reporter/blob/239e456f3f73e5abf3ad28f93be96a9b8795a9b9/index.js#L17

In fact just add `fs.readFileSync(__dirname + "/../node_modules/mocha/package.json", "utf8")` seems to work, but of course there are many way to aim that.

**Comment by YOU54F** [comment]
Is mocha 6.1.4  and mocha-junit-reporter in node_modules in the project folder (apps/site-e2e) and in which dir is your package.json that specifies them?

We could just set a flag that could be passed in, such as mocha6plus, which the user can set, which if set will override the fs check. The warning could state to the user that they can explicitly set the flag?

**Comment by gcirone** [comment]
The package json is on the project route

```
apps/site-e2e/cypress.json
package.json
```

But i think the best approach is to make it work without setting any flag, should be transparent for the user.

**Comment by YOU54F** [comment]
Agree with you mate, making it as transparent to the user as possible if preferable.

PR would be good, I can open my fork again, pull in the change and publish it as mocha-junit-reporters for testing 👌

**Comment by gcirone** [comment]
The PR is open @YOU54F https://github.com/michaelleeallen/mocha-junit-reporter/pull/88 Let me know if the solution is ok for you :)

I don't understand the `mocha-junit-reporters` why this package and what is the purpose?

**Comment by YOU54F** [comment]
Hey @gcirone solution looks good to me. I am not a maintainer of this repository and created mocha-junit-reporters so I could test the changes and allow others to consume it while we were waiting for the mocha 6 compatability fix to be merged into this repository

**Comment by gcirone** [comment]
Released in mocha-junit-reporter@1.23.1 work as expected 🥇
